01-node-tutorial/answers/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules  
+.DS_Store  
```

Comment:
You can also add a line for **/.DS_Store to remove all versions of the file to avoid committing